### PR TITLE
DEV: Remove hashtag autocomplete experimental setting

### PR DESCRIPTION
**Pull request by martin-brennan**
Everything relying on this was removed in
09223e5ae72ff1ced8f08143b7578122b462c6d6

### CODE REVIEW

**Review by tgxworld — APPROVED** [review]
Approved but wondering do we need a migration to remove the rows from the database?

**Comment by martin-brennan** [comment]
@tgxworld good call, added the migration + will wait for rebuild before I merge

**Pull request merged**
